engines/mxnet/mxnet-engine/build.gradle

```diff
@@ -55,6 +60,8 @@ tasks.register('jnarator') {
     }
 }
 
+verifyJava.dependsOn jnarator
```

Comment:
can we define all `verifyJava` dependencies in the `verifyJava` tasks definition? this would consolidate all the logic related to verifyJava in one place